tests/exec/execute.go

```diff
@@ -86,3 +92,15 @@ func ExecuteCommandOnPodWithOptions(virtCli kubecli.KubevirtClient, pod *k8sv1.P
 
 	return executor.Stream(options)
 }
+
+func CommandOnVMI(vmi *v1.VirtualMachineInstance, command string) error {
```

Comment:
You have decided to move this to a library for common usage.
As doing so, you have opened the option to discuss its correctness and fitting to the other libraries helpers.

If you do not want to have this discussion, keep this in the test file hidden and you "I just move it" argument will stand.

Reply:
Having `console.RunCommand` and `exec.CommandOnVmi` is confusing and at least I wouldn't know which one I should use when I want to run a single.command on my vmi.
The reason `console.RunCommand` is not using the safe version of the expect batch is that it is nit needed when running g a single command.
The safe batch is mostly needed for long batches to make sure the expected result is not readen from from the old buffer. console.RunCommand is written in a safe enough way for executing a singe command and it proved itself during the long time it already exist. I don't see a reason to add another "safe" way to run a simge command on the vm. The fact that the old implementation added to safe run command method was wrong since console.RunCommand had to be used. But moving the method to a common library makes this small error much bigger.

Reply:
Thanks for the discussion I appreciate it :slightly_smiling_face: 

`exec.CommanOnVMI` is removed, `console.RunComman` is being used instead.